packages/types/src/clerk.ts

```diff
@@ -837,12 +837,7 @@ export type SignInProps = RoutingOptions & {
   /**
    * Enable experimental flags to gain access to new features. These flags are not guaranteed to be stable and may change drastically in between patch or minor versions.
    */
-  __experimental?: Autocomplete<
-    {
-      newComponents: boolean;
-    },
-    Record<string, any>
-  >;
+  __experimental?: Record<string, any> & { newComponents?: boolean };
```

Comment:
This was changed because I misunderstood what the `Autocomplete` type was for. It's for autocompleting string unions, not for autocompleting arbitrary objects.